tests/migration_test.go

```diff
@@ -260,11 +260,16 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 	})
 
 	setMastersUnschedulable := func(mode bool) {
```

Comment:
Sorry for not noticing this earlier! In order to remove non inclusive word, can we also rename the function name? i.e. `setControlPlaneNodesUnschedulable`? Thanks

Reply:
@fossedihelm Of course, my bad. Done!

Reply:
@enp0s3 Thanks!